template/package.json

```diff
@@ -121,8 +122,8 @@
     "react-test-renderer": "16.13.1",
     "ts-loader": "^8.3.0",
     "typescript": "3.9.9",
-    "webpack": "5.72.0",
-    "webpack-cli": "4.9.2",
+    "webpack": "^5.72.0",
```

Comment:
please remove the ^, we use strict dependency to avoid unpredictable package updated